gensim/models/wrappers/dtmmodel.py

```diff
@@ -6,16 +6,17 @@
 # Based on Copyright (C) 2014 Radim Rehurek <radimrehurek@seznam.cz>
 
 
-"""
-Python wrapper for Dynamic Topic Models (DTM) and the Document Influence Model (DIM)  [1].
+"""Python wrapper for Dynamic Topic Models (DTM) and the Document Influence Model (DIM)  [1].
 
```

Comment:
It will be really great if you add short instruction "How to install 'backend' for it" for each wrapper.

Reply:
sure!, I forgot about it.